k8s_support.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Kubernetes support"
+nav_order: 9
+description: "Deploy FalkorDB to Kubernetes."
+---
+
+## Kubernetes support for FalkorDB
+
+FalkorDB can be deployed on Kubernetes using Helm charts and Docker images. This guide will walk you through the process.
+
+### Prerequisites
+
+Before you begin, make sure you have Helm installed on your Kubernetes cluster.
+
+To deploy FalkorDB to Kubernetes we need to use:
+
+* [Helm charts](https://bitnami.com/stack/redis/helm)
+* [Docker image](https://hub.docker.com/r/falkordb/falkordb)
+
+And follow this steps:
+
+### Step 1: Create a `values.yaml` File
+
+1.Create a values.yaml file with the following content:
+
+```yaml
+image:
+  registry: docker.io
+  repository: falkordb/falkordb
+  tag: "4.0"
+
+master:
+  extraFlags:
+  - "--loadmodule /FalkorDB/bin/linux-x64-release/src/falkordb.so"
+
+slave:
+  extraFlags:
+  - "--loadmodule /FalkorDB/bin/linux-x64-release/src/falkordb.so"
+```
+This file specify the FalkorDB image(you can choose different tags)
+and configure the master and slave to load the FalkorDB module.
+For additional configurations [see the official Helm chart documentation](https://github.com/bitnami/charts/blob/main/bitnami/redis/values.yaml)
+
+### Step 2: Install FalkorDB Helm Charts
+Install the helm charts using the following command:
+
+```bash
+helm install -f values.yaml my-falkordb oci://registry-1.docker.io/bitnamicharts/redis
+```
+
+This command deploys FalkorDB with the configuration from values.yaml.
+After running this command, instructions on how to connect to the FalkorDB server will be displayed.
+
+### Step 3: Retrieve the FalkorDB Password
+
+To connect to FalkorDB, you need the Redis password. Retrieve it using the following command:
+
+```bash
+export REDIS_PASSWORD=$(kubectl get secret --namespace default my-release-redis -o jsonpath="{.data.redis-password}" | base64 -d)
+```
```

Comment:
I'll assume that if we run our own Helm chart we'll have full control over which password is used.

Reply:
there is a password parameter in this charts

Reply:
Right, I'll assume we can have full control over the password if we had our own chart